### PR TITLE
[AD-144] Creating new data_source and pointing most dismissal metrics to it. Also new metrics for organic tile dismissals.

### PR DESCRIPTION
**Pull request by sergiosonline**
Creating new data_source `[data_sources.newtab_visits_topsite_tile_interactions]` based on `newtab_visits`. Pointing most dismissal metrics to it. Also adding metrics for organic tile dismissals.

### CODE REVIEW

File: definitions/firefox_desktop.toml [hunk 1]
```diff
@@ -1278,36 +1279,51 @@ friendly_name = "Sponsored Tiles Disabled Event"
...
+        0
       )"""
-data_source = "activity_stream_events"
+data_source = "newtab_visits_topsite_tile_interactions"
 friendly_name = "Sponsored Tiles Dismissals Count (Positions 1 and 2)"
 description = "Count of sponsored tiles dismissals in the first two positions"
 
+[metrics.sponsored_tiles_dismissals_pos3_more]
+select_expression = """COALESCE(
+        SUM(CASE WHEN topsite_tile_interactions.sponsored_topsite_tile_dismissals > 0 AND topsite_tile_interactions.topsite_tile_position >= 3 THEN topsite_tile_interactions.sponsored_topsite_tile_dismissals ELSE 0 END),
```
Comment (m-d-bowerman) [inline definitions/firefox_desktop.toml:1306]: Tile positions in `newtab` are 0-indexed.

**Review by m-d-bowerman — CHANGES_REQUESTED** [review]
Only left one comment, but I think every reference you have to position needs to be updated to reflect the 0-index for tile positions.

File: definitions/firefox_desktop.toml (hunk 1 above)
Comment (sergiosonline) [reply definitions/firefox_desktop.toml]: Thanks for the callout. Updated!

File: definitions/firefox_desktop.toml [hunk 2]
```diff
@@ -1554,14 +1570,28 @@ experiments_column_type = "native"
...
+        topsite_tile_interactions
+    FROM
+        `moz-fx-data-shared-prod.telemetry.newtab_visits` e
+    CROSS JOIN
+        UNNEST(e.topsite_tile_interactions) AS topsite_tile_interactions
+)"""
+submission_date_column = "submission_date"
+description = "Topsite Tiles Visit Activity Daily"
+friendly_name = "Topsite Tiles Visit Activity Daily"
+
```
Comment (m-d-bowerman) [inline definitions/firefox_desktop.toml:1594]: add `client_id_column = 'legacy_telemetry_client_id`

**Review by m-d-bowerman — CHANGES_REQUESTED** [review]

**Pull request merged**